public/video-ui/src/components/FormFields/RichTextField.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import RequiredForComposer from '../../constants/requiredForComposer';
+import { EditorConfig } from './richtext/config';
+import { getWords, isTooLong } from './richtext/utils/richTextHelpers';
+import { RichTextEditor } from './RichTextEditor';
+
+type EditorState = {
+  wordCount: number;
+  copiedValue: {
+    text: string | null;
+    seed: number | null;
```

Comment:
Am I right in thinking that this is used more as an `updateID` than `seed`?

Reply:
Yeah - though as alluded above I hope we can get rid of it completely.